modules/nodes-nodes-resources-configuring-about.adoc

```diff
@@ -98,16 +67,12 @@ before reaching out of memory conditions are not available for pods.
 The following is an example to illustrate the impact of node allocatable for memory:
 
 * Node capacity is `32Gi`
-* --kube-reserved is `2Gi`
-* --system-reserved is `1Gi`
+* --system-reserved is `3Gi`
 * --eviction-hard is set to `100Mi`.
 
-For this node, the effective node allocatable value is `28.9Gi`. If the node
-and system components use up all their reservation, the memory available for pods is `28.9Gi`,
-and kubelet will evict pods when it exceeds this usage.
+For this node, the effective node allocatable value is `28.9Gi`. If the node and system components use all their reservation, the memory available for pods is `28.9Gi`, and `kubelet` evicts pods when it exceeds this threshold.
 
-If you enforce node allocatable (`28.9Gi`) via top level cgroups, then pods can never exceed `28.9Gi`.
-Evictions would not be performed unless system daemons are consuming more than `3.1Gi` of memory.
+If you enforce node allocatable, `28.9Gi`, with top-level cgroups, then pods can never exceed `28.9Gi`. Evictions are not performed unless system daemons consume more than `3.1Gi` of memory.
```

Comment:
Any thoughts on what this means _with top-level cgroups_?
Is this a reference to using `system-reserved-cgroup`?

Reply:
I'm not very clear on what a lot of this area covers.  Some guesses:
* I suspect the words are old, like from 1.6-ish of k8s, and have rotten a bit.
* I think `/system.slice` is used by default--assuming I understood what I saw after cat /etc/kubernetes/kubelet.conf.
* So, vanilla k8s might not use cgroups.  IIUC, OCP 4 uses /system.slice for system processes and /kubepods.slice for pods.

I might be able to dig deeper, but I need names of engineering contacts who can help answer questions.

Reply:
@soltysh reviewed this topic for me when I moved it from 3.11 to 4.x. Perhaps he can help here?

Reply:
@rphillips will be a better choice here.

Reply:
The information is not readily available in my head and getting it out of @rphillips head isn't scheduled.  My instinct is to limit the updates to the BZ and further refinements can be a different PR.  That said, if SMEs feel strongly and have time to help me enhance this area, I will.